### PR TITLE
Fix general proofreading

### PR DESCRIPTION
**Pull request by ottok**
Preferred reviewer: @anttiviljami

### CODE REVIEW

File: roles/scripts/files/wp-list-env [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+##
+# Gives user easy way to environment variables which are important for him
+##
+
+# Source environment if doesn't exist yet
+if [ -z $PS1 ]; then
+  source /etc/container_environment.sh
+fi
+
+IFS='%' read -a envvars <<< "$LIST_CONTAINER_ENVS"
```
Comment (anttiviljami) [inline roles/scripts/files/wp-list-env]: is this tested? i don't think the vagrant box has /etc/container_environment since it's not a container

**Comment by ottok** [comment]
I didn't test anything. I don't know how to rebuild the Vagrant image. Please review carefully. Thanks!

**Comment by anttiviljami** [comment]
Yeah, can you revert adding the wp-list-env script? It has to be implemented differently

File: roles/scripts/files/wp-list-env (hunk 1 above)
Comment (ottok) [inline roles/scripts/files/wp-list-env]: The file is there and the contents looks sane. It is probably used somewhere. What do wp-list-env script does will not affect anything else than make users able to run this command as promised in docs.wp-palvelu.fi

```
vagrant@wp-dev:~$ ll /etc/conta*
-rw-rw-r-- 1 vagrant vagrant 387 Feb 22 10:20 /etc/container_environment.sh
vagrant@wp-dev:~$ cat /etc/container_environment.sh 
# Just some helpers to make this look like the production environment
export DB_NAME=vagrant
export DB_USER=vagrant
export DB_PASSWORD=vagrant
export WP_PROJECT=/data/wordpress
export HTDOCS=$WP_PROJECT/htdocs
export WEB_ROOT=$HTDOCS
export WP_CORE=$HTDOCS/wordpress
export WP_CONTENT=$HTDOCS/wp-content
export WP_ENVIRONMENT=development

export HTTPS_DOMAIN_ALIAS=pointscene.seravo.dev

```

Comment (anttiviljami) [inline roles/scripts/files/wp-list-env]: ok, cool. i guess this would work fine then.

**Comment by ottok** [comment]
The container_environment.sh file is there and the contents looks sane. It is probably used somewhere. What the wp-list-env script does will not affect anything else than make users able to run this command as promised in docs.wp-palvelu.fi. It would not make sense to revert that commit, as it really fixes an issue and is very unlikely to have any unforseen effects (only people who run wp-list-env are affected, now they get something instead of a file-not-found error).

```
vagrant@wp-dev:~$ ll /etc/conta*
-rw-rw-r-- 1 vagrant vagrant 387 Feb 22 10:20 /etc/container_environment.sh
vagrant@wp-dev:~$ cat /etc/container_environment.sh 
# Just some helpers to make this look like the production environment
export DB_NAME=vagrant
export DB_USER=vagrant
export DB_PASSWORD=vagrant
export WP_PROJECT=/data/wordpress
export HTDOCS=$WP_PROJECT/htdocs
export WEB_ROOT=$HTDOCS
export WP_CORE=$HTDOCS/wordpress
export WP_CONTENT=$HTDOCS/wp-content
export WP_ENVIRONMENT=development

export HTTPS_DOMAIN_ALIAS=pointscene.seravo.dev
```

**Pull request merged**